c172p.xml

```diff
@@ -1099,6 +1099,17 @@
             </product>
         </function>
 
+        <function name="fcs/aileron-pos-rad-avgd">
+            <description>Averaged aileron position</description>
+            <product>
```

Comment:
There's a `<avg>` element that you can use instead. See section 2.5.1 of the JSBSim Reference Manual.

Reply:
Done. (lines 1102 -1111)
Is there a more elegant way to do it? With this writing (because of the minus sign), is has the same number of lines as my initial code...